client/src/features/garden/Cesium.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable quotes */
+import "./garden.scss";
+import { Component } from "react";
+import {
+  Viewer,
+  Ion,
+  createOsmBuildings,
+  Cartesian3,
+  Math,
+  createWorldImagery,
+  IonWorldImageryStyle,
+  IonImageryProvider
+} from "cesium";
+import "cesium/Build/Cesium/Widgets/widgets.css";
+window.CESIUM_BASE_URL = "https://cesiumjs.org/";
+
+export class Cesium extends Component {
```

Comment:
We would prefer this to be a functional component if possible, will look into it

Reply:
fixed in bd64fdd